ui/src/websites/artconomy/ArtconomyLogin.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { LoginDialogProps } from '../interfaces/website.interface';
+import { Form, Input, Button, message } from 'antd';
+import LoginService from '../../services/login.service';
+import Axios, {AxiosError} from 'axios';
+
+interface State {
+    email: string;
+    password: string;
+    token: string;
+    show2fa: boolean,
+    sending: boolean;
+}
+
+export default class ArtconomyLogin extends React.Component<LoginDialogProps, State> {
+    state: State = {
+        email: '',
+        password: '',
+        token: '',
+        show2fa: false,
+        sending: false,
+    };
+
+    constructor(props: LoginDialogProps) {
+        super(props);
+        this.state = {
+            ...this.state,
+            ...(props.data as State)
+        };
+    }
+
+    async submit() {
+        this.setState({ sending: true });
+        const auth = await Axios.post<{ id: number, username: string, csrftoken: string, token: string[] }>(
+            `${window.electron.environment.ARTCONOMY_URL}/api/profiles/v1/login/`,
+            {...this.state},
+            { responseType: 'json' },
+        ).catch((error: AxiosError) => {
+            if (!error.response) {
+                message.error("Unable to contact Artconomy's servers. Are you online?")
```

Comment:
```suggestion
                message.error('Unable to contact Artconomy's servers. Are you online?')
```

Reply:
There's an apostrophe in this string-- would you prefer I go to singlequote and use a backslash or keep it as is?